engine/table/src/test/java/io/deephaven/engine/table/impl/updateby/TestCumSum.java

```diff
@@ -211,91 +211,7 @@ protected Table e() {
      */
     // endregion
 
-    private long[] cumsum(byte[] values) {
-        if (values == null) {
-            return null;
-        }
-
-        if (values.length == 0) {
-            return new long[0];
-        }
-
-        long[] result = new long[values.length];
-        result[0] = isNull(values[0]) ? NULL_LONG : values[0];
-
-        for (int i = 1; i < values.length; i++) {
-            final boolean curValNull = isNull(values[i]);
-            if (isNull(result[i - 1])) {
-                result[i] = curValNull ? NULL_LONG : values[i];
-            } else {
-                if (curValNull) {
-                    result[i] = result[i - 1];
-                } else {
-                    result[i] = result[i - 1] + values[i];
-                }
-            }
-        }
-
-        return result;
-    }
-
-    private long[] cumsum(short[] values) {
-        if (values == null) {
-            return null;
-        }
-
-        if (values.length == 0) {
-            return new long[0];
-        }
-
-        long[] result = new long[values.length];
-        result[0] = isNull(values[0]) ? NULL_LONG : values[0];
-
-        for (int i = 1; i < values.length; i++) {
-            final boolean curValNull = isNull(values[i]);
-            if (isNull(result[i - 1])) {
-                result[i] = curValNull ? NULL_LONG : values[i];
-            } else {
-                if (curValNull) {
-                    result[i] = result[i - 1];
-                } else {
-                    result[i] = result[i - 1] + values[i];
-                }
-            }
-        }
-
-        return result;
-    }
-
-    private long[] cumsum(int[] values) {
-        if (values == null) {
-            return null;
-        }
-
-        if (values.length == 0) {
-            return new long[0];
-        }
-
-        long[] result = new long[values.length];
-        result[0] = isNull(values[0]) ? NULL_LONG : values[0];
-
-        for (int i = 1; i < values.length; i++) {
-            final boolean curValNull = isNull(values[i]);
-            if (isNull(result[i - 1])) {
-                result[i] = curValNull ? NULL_LONG : values[i];
-            } else {
-                if (curValNull) {
-                    result[i] = result[i - 1];
-                } else {
-                    result[i] = result[i - 1] + values[i];
-                }
-            }
-        }
-
-        return result;
-    }
-
-    private long[] cumsum(Boolean[] values) {
+    private long[] boolean_cumsum(Boolean[] values) {
```

Comment:
Does the query library need a boolean cumsum?  I'm all for waiting on a user request.

Reply:
`Boolean` ops from `Numeric.java` are easily computed with the help of `ifelse()` from `Basic.java` library as well, so unless we see a clear performance problem we may never need it.